lib/src/entrypoint.dart

```diff
@@ -936,3 +1065,9 @@ bool detectWindowsLineEndings(String text) {
   }
   return windowsNewlines > unixNewlines;
 }
+
+class _ResolutionSuggestion {
+  final String suggestion;
+  final int priority;
```

Comment:
* List of package names affected
 * Number of upgrades
 * Number of downgrades